src/source/PeerConnection/Rtcp.c

```diff
@@ -25,3 +30,43 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
 
     return retStatus;
 }
+
+STATUS handleRemb(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerConnection)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PUINT32 pSsrcList = NULL;
+    DOUBLE maximumBitRate = 0;
+    UINT8 ssrcListLen;
+    UINT32 i;
+    PDoubleListNode pCurNode = NULL;
+    PKvsRtpTransceiver pTransceiver = NULL;
+    UINT64 item;
+
+    CHK_STATUS(rembValueGet(pRtcpPacket->payload, pRtcpPacket->payloadLength, &maximumBitRate, NULL, &ssrcListLen));
+    CHK(NULL != (pSsrcList = (PUINT32) MEMCALLOC(1, SIZEOF(UINT32) * ssrcListLen)), STATUS_STORE_OUT_OF_MEMORY);
+    CHK_STATUS(rembValueGet(pRtcpPacket->payload, pRtcpPacket->payloadLength, &maximumBitRate, pSsrcList, &ssrcListLen));
+
+    for (i = 0; i < ssrcListLen; i++) {
+        CHK_STATUS(doubleListGetHeadNode(pKvsPeerConnection->pTransceievers, &pCurNode));
+        while(pCurNode != NULL && pTransceiver == NULL) {
+            CHK_STATUS(doubleListGetNodeData(pCurNode, &item));
+            CHK(item != 0, STATUS_INTERNAL_ERROR);
+
+            pTransceiver = (PKvsRtpTransceiver) item;
+            if (pTransceiver->sender.ssrc != pSsrcList[i]) {
+                pTransceiver = NULL;
+            }
+
+            pCurNode = pCurNode->pNext;
+        }
+        CHK_ERR(pTransceiver != NULL, STATUS_RTCP_INPUT_SSRC_INVALID, "Receiving REMB for non existing ssrcs: ssrc %lu", pSsrcList[i]);
+        if (pTransceiver->onBandwidthEstimation != NULL) {
```

Comment:
Do the check initially and don't go into the function and bail out if the callback is NULL